src/cli.rs

```diff
@@ -761,8 +764,255 @@ pub fn run(sub: Subcommand, no_manifest_update: bool, try_manifest_update: bool)
                 report_cloud_changes(&changes, api);
             }
         },
-    }
+        Subcommand::Wrap {
+            name_source,
+            gui,
+            commands,
+        } => 'wrap: {
+            log::info!(
+                "WRAP::setup: infer: {:#?}, name: {:#?}, gui: {:#?}, commands: {:#?}",
+                name_source.infer,
+                name_source.name,
+                gui,
+                commands
+            );
+            failed = true;
+            // TODO.2023-07-17 pass legendary path downwards
+            // Determine game name
+            //
+            let mut game_name;
+            if let Some(name) = name_source.name {
+                game_name = name;
+            } else {
+                let roots = config.expanded_roots();
+                match name_source.infer.unwrap() {
+                    parse::LauncherTypes::Heroic => {
+                        match get_game_name_from_heroic_launch_commands(&roots, &commands) {
+                            Ok(name) => game_name = name,
+                            Err(msg) => {
+                                let _ = native_dialog::MessageDialog::new()
+                                .set_title("Ludusavi Wrap Error")
+                                .set_text(
+                                    &format!("Could not determine game name from launch commands, aborting. \n\n Error message: {:?}", msg))
+                                .set_type(native_dialog::MessageType::Error)
+                                .show_alert();
+                                break 'wrap;
+                            }
+                        }
+                    }
+                }
+            }
+            log::debug!("WRAP::setup: game name is: {}", game_name);
+
+            //
+            // Check using TitleFinder
+            //
+            // e.g. "Slain: Back From Hell" from legendary to "Slain: Back from
+            // Hell" as known to ludusavi
+            //
+            let manifest = load_manifest(&config, &mut cache, no_manifest_update, try_manifest_update)?;
+            let title_finder = TitleFinder::new(
+                &manifest,
+                &BackupLayout::new(config.restore.path.clone(), config.backup.retention.clone()),
+            );
+            match title_finder.find_one(&[normalize_title(&game_name)], &None, &None, true, false, true) {
```

Comment:
Ic.  I didn't get the meaning of `find_one`s parameters since they are not documented.
Could you be so kind and describe them in the source code?
And thanks for the hint!

Reply:
> Could you be so kind and describe them in the source code?

Yeah, I should really add some documentation throughout the code base 😅 I'll try to improve that when I can.